pkg/kubelet/volumemanager/reconciler/reconciler_test.go

```diff
@@ -60,6 +60,40 @@ const (
 
 func hasAddedPods() bool { return true }
 
+func Test_UpdateStates(t *testing.T) {
+	// Arrange
+	volumePluginMgr, _ := volumetesting.GetTestVolumePluginMgr(t)
+	dsw := cache.NewDesiredStateOfWorld(volumePluginMgr)
+	asw := cache.NewActualStateOfWorld(nodeName, volumePluginMgr)
+	kubeClient := createTestClient()
+	fakeRecorder := &record.FakeRecorder{}
+	fakeHandler := volumetesting.NewBlockVolumePathHandler()
+	oex := operationexecutor.NewOperationExecutor(operationexecutor.NewOperationGenerator(
+		kubeClient,
+		volumePluginMgr,
+		fakeRecorder,
+		false, /* checkNodeCapabilitiesBeforeMount */
+		fakeHandler,
+	))
+	recon := NewReconciler(
+		kubeClient,
+		false, /* controllerAttachDetachEnabled */
+		reconcilerLoopSleepDuration,
+		waitForAttachTimeout,
+		nodeName,
+		dsw,
+		asw,
+		hasAddedPods,
+		oex,
+		mount.NewFakeMounter(nil),
+		hostutil.NewFakeHostUtil(nil),
+		volumePluginMgr,
+		kubeletPodsDir)
+	rec := recon.(*reconciler)
+	rec.kubeClient = nil
+	rec.updateDevicePath(nil)
```

Comment:
/assign @jingxu97 
to see if there's a better way to test reconstruction in unit tests

Reply:
Instead of nullifying rec.kubeClient on line 93, we can pass nil for parameter kubeClient of the NewReconciler call.
Effect would be the same.